app/components/primer/alpha/banner.rb

```diff
@@ -61,21 +61,24 @@ class Banner < Primer::Component
         DEFAULT_SCHEME => "",
         :warning => "Banner--warning",
         :danger => "Banner--error",
-        :success => "Banner--success"
+        :success => "Banner--success",
+        :upsell => "Banner--upsell"
       }.freeze
 
       LEGACY_SCHEME_MAPPINGS = {
         DEFAULT_SCHEME => "",
         :warning => "flash-warn",
         :danger => "flash-error",
-        :success => "flash-success"
+        :success => "flash-success",
+        :upsell => "flash-upsell",
```

Comment:
```suggestion
```

Adding this `:upsell` option to the legacy scheme mappings is likely not necessary.